hack/generator/pkg/config/type_transformer.go

```diff
@@ -16,7 +16,9 @@ import (
 
 // A transformation target
 type TransformTarget struct {
-	PackagePath string `yaml:",omitempty"`
+	LibraryPath string `yaml:",omitempty"`
```

Comment:
I don't think having this here makes sense because LibraryPath (as per your language above) was for dependencies that aren't GVK's, and the whole point of `TransformTarget` is that it's transforming types we're generating, which by definition have a GVK (and thus aren't a member of a "library")?

I think you just want Group+Version here?

Reply:
After checking with @babbageclunk, agreed.